packages/compass-home/src/components/home.tsx

```diff
@@ -378,6 +378,7 @@ function getCurrentTheme(): Theme {
 function ThemedHome(
   props: React.ComponentProps<typeof Home>
 ): ReturnType<typeof Home> {
+  const preferences = preferencesLocator();
```

Comment:
Let's maybe leave a todo here 😅 

```suggestion
  // TODO(COMPASS-7433): should not be used directly in render, will go away with a home cleanup
  const preferences = preferencesLocator();
```

Reply:
Yup, added in 99493319f935f878cdbb40f792a1835f5986e977 :+1: